src/components/shop/shop.js

```diff
@@ -69,6 +69,7 @@ export class Shop extends Screen {
         Object.keys(this.panes).forEach(key =>
             pane === key ? this.panes[key].setVisible(true) : this.panes[key].setVisible(false),
         );
+        this.title.setTitleText(pane === "top" ? "Shop" : pane);
```

Comment:
That hard-coded string probably needs to go, but have left this for future tix happening around the titles.